Automattic/studio
# Downgrade Electron Forge to version `7.3.1`

This PR is a follow-up of https://github.com/Automattic/studio/pull/198 to address a failure on Windows when generating the builds (https://github.com/Automattic/studio/pull/135#issuecomment-2144811867).

## Proposed Changes

- Downgrade Electron Forge packages to version `7.3.1` to avoid using the version `>= 18.2.0` of the package`electron/packager`. This is mainly to avoid [this change](https://github.com/electron/packager/commit/d9655d4bab5a9328391e24c98235ef0d241f1e71) in version `7.4.0` that requires that build versions are in number format (e.g. `a.b.c.d`).

## Testing Instructions



- Observe that all CI jobs pass.
- Observe that builds work as expected in the different architectures.

## Pre-merge Checklist
